apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: CTABanner
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0'
+source: https://github.com/primer/brand/tree/main/src/CTABanner/CTABanner.tsx
+storybook: '/brand/storybook/?path=/docs/components-ctabanner--playground'
+description: Use CTABanner to create call to action content in a separated section.
+---
+
+```js
+import {CTABanner, ButtonGroup, Button} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTABanner>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Aligned center
+
+The content alignment can be changed using the `align` prop on the root `<ComparisonTable>`. This can be configured to either `start` or `center`.
```

Comment:
ComparisonTable? 😅 

```suggestion
The content alignment can be changed using the `align` prop on the root `CTABanner`. This can be configured to either `start` or `center`.
```